packages/core-mobile/package.json

```diff
@@ -16,12 +16,14 @@
     "postinstall": "node ${PROJECT_CWD}/scripts/generate-metro-monorepo-config.js; node_modules/.bin/patch-package; node_modules/.bin/rn-nodeify --install events,stream,vm,assert,https,http,os,zlib,path,fs --hack; node scripts/codegen.js",
     "storybook-generate": "sb-rn-get-stories --v6-store",
     "storybook-watch": "sb-rn-watcher",
+    "codegen-old": "mkdir -p app/utils/apiClient/generated && yarn gen:contracts && yarn gen:openapi-zod:profileApi && yarn gen:openapi-zod:balanceApi",
     "gen:contracts": "typechain --target=ethers-v6 --out-dir app/contracts/openzeppelin ./node_modules/@openzeppelin/contracts/build/contracts/ERC20.json ./node_modules/@openzeppelin/contracts/build/contracts/ERC721.json ./node_modules/@openzeppelin/contracts/build/contracts/ERC1155.json",
-    "gen:glacierApi": "npx openapi-zod-client 'https://glacier-api-dev.avax.network/api-json' -o './app/utils/apiClient/generated/glacierApi.client.ts'",
-    "gen:tokenAggregatorApi": "npx openapi-zod-client 'https://core-token-aggregator.avax-test.network/schema.json' -o './app/utils/apiClient/generated/tokenAggregatorApi.client.ts'",
-    "gen:profileApi": "npx openapi-zod-client 'https://core-profile-api.avax-test.network/schema.json' -o './app/utils/apiClient/generated/profileApi.client.ts' && node ./app/utils/apiClient/scripts/fixZodIntersections.js ./app/utils/apiClient/generated/profileApi.client.ts",
-    "gen:balanceApi": "npx @hey-api/openapi-ts -f balance-api.config.js",
-    "codegen": "mkdir -p app/utils/apiClient/generated && yarn gen:contracts && yarn gen:glacierApi && yarn gen:tokenAggregatorApi && yarn gen:profileApi && yarn gen:balanceApi",
+    "gen:openapi-zod:profileApi": "npx openapi-zod-client 'https://core-profile-api.avax-test.network/schema.json' -o './app/utils/apiClient/generated/profileApi.client.ts' && node ./app/utils/apiClient/scripts/fixZodIntersections.js ./app/utils/apiClient/generated/profileApi.client.ts",
+    "gen:openapi-zod:balanceApi": "npx @hey-api/openapi-ts -i 'https://core-balance-api.avax-test.network/schema.json' -o './app/utils/apiClient/generated/balanceApi.client'",
+    "gen:openapi:glacier": "npx openapi-typescript 'https://glacier-api-dev.avax.network/api-json' -o './app/utils/api/generated/glacier/schema.d.ts'",
+    "gen:openapi:tokenAggregator": "npx openapi-typescript 'https://core-token-aggregator.avax-test.network/schema.json' -o './app/utils/api/generated/tokenAggregator/schema.d.ts'",
+    "gen:openapi:all": "yarn gen:openapi:glacier && yarn gen:openapi:tokenAggregator",
+    "codegen": "yarn gen:openapi:all && yarn codegen-old",
```

Comment:
we have a codegen.js script now and we run it postinstall. could you remove this `codegen` and `codegen-old` command and use that script instead?